test/network-imports/vitest.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    poolOptions: {
+      threads: {
+        execArgv: ['--experimental-network-imports'],
+      },
+      forks: {
+        execArgv: ['--experimental-network-imports'],
+      },
+      // not supported?
+      //   FAIL  test/basic.test.ts [ test/basic.test.ts ]
+      //   Error: ENOENT: no such file or directory, open 'http://localhost:9602/slash@3.0.0.js'
+      //    ❯ Object.openSync node:fs:596:3
+      //    ❯ readFileSync node:fs:464:35
+      vmThreads: {
+        execArgv: ['--experimental-network-imports'],
+      },
```

Comment:
It looks like it's not working on vmThreads. I'll investigate further to see if it's simply NodeJS limitation.

Reply:
I think every import is delegated to our VM implementation, and it just doesn't support `http`.

Reply:
Thanks for the hint! I haven't had a chance to look at how vmThreads works yet, so I'll look around a bit to get the idea.
I suppose it's still okay even if it doesn't work on vmThreads?

Reply:
Yes, we can disable this test for vm threads/forks.

I think it's a separate feature request (maybe create an issue for it). Shouldn't be hard to implement.